### ISSUE TITLE
Space creation flow makes poor use of space

### ISSUE THREAD

**Issue by kittykat** [vector-im/element-web#21589]
#### What would you like to do?
Quickly and effectively browse rooms that I am a member of

#### Why would you like to do it?
To set up a private space for organising my rooms

#### How would you like to achieve it?
By having "what do you want to organise?" view make good use of the available space.

### Additional context

![Screenshot from 2022-03-29 10-10-04](https://user-images.githubusercontent.com/51663/160585090-9dc765fa-140b-464f-bd88-96c5c5da762b.png)

At the moment, most of the screen is unused, while I am in dozens of rooms which I have to browse through a small scrollable area. The area could potentially expand up to window height before it becomes scrollable so that I can see more rooms at the same time.

**Comment by Sinharitik589** [comment]
@gsouquet  can you assign it to me.I am working on it.

**Comment by gsouquet** [comment]
@Sinharitik589 I don't think this is ready to get worked on, we first need to get input from the design team

**Comment by Jumeb** [comment]
Hello, @andybalaam I would love to work on this issue, could you please assign it to me.
